resources/schemas/dbscripts/postgresql/targetedms-19.14-19.15.sql

```diff
@@ -31,6 +31,7 @@ CREATE TABLE targetedms.AuditLogEntry (
 );
 
 CREATE UNIQUE INDEX uix_auditLogEntry_document on targetedms.AuditLogEntry USING btree (documentGUID, entryHash);
+CREATE INDEX uix_auditLogEntry_version on targetedms.AuditLogEntry USING btree (versionId);
```

Comment:
A LabKey deployment won't re-run a script that it's already executed. As such, dev machines that are already upgraded to 19.15 or later won't end up creating this index.

It can be added as a new SQL script and a separate version number bump. Unfortunately, we don't have a great solution for how to manage multiple active feature branches with schema updates, and this PR also has upgrades:

https://github.com/LabKey/targetedms/pull/4

Can you update this branch so that this index creation happens in a version update from 19.191-19.192? We will get the other PR merged first (which will get the version to 19.191) and then merge this one.